roottest/root/io/compression/CMakeLists.txt

```diff
@@ -1,7 +1,47 @@
-#-------------------------------------------------------------------------------
-#
-# Placeholder file to translate the tests to the new CTest system. Meanwhile we
-# define a CTest test that calls 'make' in ${CMAKE_CURRENT_SOURCE_DIR}
-#
-#-------------------------------------------------------------------------------
-ROOTTEST_ADD_OLDTEST()
+ROOT_GENERATE_DICTIONARY(G__IoCompressionGeneration
+  ${CMAKE_CURRENT_SOURCE_DIR}/Event.h
+  LINKDEF ${CMAKE_CURRENT_SOURCE_DIR}/EventLinkDef.h)
+
+ROOT_LINKER_LIBRARY(IoCompressionGeneration
+  ${CMAKE_CURRENT_SOURCE_DIR}/IoCompressionGeneration.cxx
+  G__IoCompressionGeneration.cxx
+  LIBRARIES Core Tree Hist MathCore Physics Graf Matrix)
```

Comment:
From https://github.com/root-project/roottest/pull/774#discussion_r2020252931:

I recommend adding the `ROOT_GENERATE_DICTIONARY` as a `FIXTURES_REQUIRED`

Reply:
There is `ROOTTEST_GENERATE_DICTIONARY` macro which supports `FIXTURES_REQUIRED`.
Example can be found in `roottest/root/io/json` sub-dir